src/services/fetchApi.jsx

```diff
@@ -0,0 +1,18 @@
+export const fetchApi = (url, method, body) => {
+  if(body) {
+    return fetch(url, {
+      method,
+      header: {
+        'Content-Type': 'application/json'
```

Comment:
need some conditional logic here. you only want to send content-type and body sometimes